openfe/protocols/openmm_afe/base.py

```diff
@@ -289,7 +289,7 @@ def _get_modeller(
         solvent_component: Optional[SolventComponent],
         smc_components: dict[SmallMoleculeComponent, OFFMolecule],
         system_generator: SystemGenerator,
-        solvation_settings: SolvationSettings
+        solvation_settings: BaseSolvationSettings
```

Comment:
Using BaseSolvation settings here because we can expect derivatives of this base class to have different solvation setttings that aren't OpenMMSolvationSettings